apps/test/integration-tests.js

```diff
@@ -5,13 +5,15 @@ import {
   throwOnConsoleWarningsEverywhere
```

Comment:
Our integration tests hit a bunch of code that tries to log metrics events to Firehose.  This doesn't actually hit the firehose service, but it did result in a lot of log noise in our test output.  Stubbing the firehose logging events in integration tests cleans up log output a bunch and will make it easier to diagnose test failures in the future.